uportal-war/src/main/resources/org/jasig/portal/portlets/SharedParameters.cpd.xml

```diff
@@ -33,54 +33,40 @@
         <parameter>
             <name>iconUrl</name>
             <label>display.icon.url</label>
+            <description>URL of the icon to use in the standard portlet</description>
             <single-text-parameter-input display="text"/>
         </parameter>
 
         <parameter>
             <name>mobileIconUrl</name>
             <label>display.mobile.icon.url</label>
+            <description>URL of the icon to use on Mobile devices</description>
             <single-text-parameter-input display="text"/>
         </parameter>
 
         <parameter>
-            <name>showChrome</name>
-            <label>show.portlet.window.chrome</label>
-            <single-choice-parameter-input display="select">
-                <default>true</default>
-                <option value="false" label="false"/>
-                <option value="true" label="true"/>
-            </single-choice-parameter-input>
-        </parameter>
-
-        <parameter>
             <name>alternativeMaximixedLink</name>
             <label>open.alternative.url.in.maximized.view</label>
+            <description>Alternate URL to display when the portlet is in MAXIMIZED mode.</description>
             <single-text-parameter-input display="text"/>
         </parameter>
 
         <parameter>
-            <name>alternate</name>
-            <label>use.alternate.appearance</label>
-            <single-choice-parameter-input display="select">
-                <default>false</default>
-                <option value="false" label="false"/>
-                <option value="true" label="true"/>
-            </single-choice-parameter-input>
-        </parameter>
```

Comment:
Can I assume this item was dealt with?  The 'alternate' options seems to be handled in content.xsl exactly as the other styles are -- namely, you can set it the old way (alternate=true) or the new way (chromeStyle=alternate).

## ToDo
- [ ] Create a JIRA for enhancing import to transform the old style into the new, so we don't have the old style in the DB any longer